### PR TITLE
Introduction to Express.js

### PR DESCRIPTION
**Pull request by avishekjana**

### CODE REVIEW

File: backend-dev-with-express/introduction-to-express/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Text
+In this lesson, we will create an HTTP server using Express.js.
+
+Express.js is a fast and lightweight framework, used majorly for web application development. Express is built on top of Node.js. It is designed to develop websites, web apps, & API’s easily.
+With that being said, let's create our first ever Express application.
+
+### Installing Express.js
+To install Express.js, first, we have to create a project folder and then a `package.json` file, which will be holding all the project dependencies. So, create a folder using `mkdir`, and inside the folder, initialize node using the following command:
```
Comment (vikispeaks) [inline backend-dev-with-express/introduction-to-express/README.md:8]: We can remove the npm init part as we have already introduced the same in the level 2. We can directly start from express installation.

File: backend-dev-with-express/introduction-to-express/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Then, open the project folder in your favourite editor, and check the newly generated `package.json.file`.
+
+Now, to install the express.js in this application, run the following command:
+````
+npm install express --save
+````
+
+### Configuring the Express server
+Node.js has a built-in `http` module as well as `https` module, to create an HTTP/HTTPS server. But the implementation process is kind of difficult.
+Whereas, Express wraps the built-in `http` module and provide a cleaner interface to create an HTTP web server.
```
Comment (vikispeaks) [inline backend-dev-with-express/introduction-to-express/README.md:22]:
```suggestion
Whereas, Express wraps the built-in `http` module and provides a cleaner interface to create an HTTP web server.
```

File: backend-dev-with-express/introduction-to-express/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+First create a new file called `index.js`, there we will create and configure our Express.JS server.
```
Comment (vikispeaks) [inline backend-dev-with-express/introduction-to-express/README.md:24]:
```suggestion
First, create a new file called `index.js`, there we will create and configure our Express.js server.
```

File: backend-dev-with-express/introduction-to-express/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+### Installing Express.js
+To install Express.js, first, we have to create a project folder and then a `package.json` file, which will be holding all the project dependencies. So, create a folder using `mkdir`, and inside the folder, initialize node using the following command:
+````
+npm init
+````
+The generator will ask you a few questions, it’s OK to just press [enter] for all of them.
+Then, open the project folder in your favourite editor, and check the newly generated `package.json.file`.
+
+Now, to install the express.js in this application, run the following command:
```
Comment (vikispeaks) [inline backend-dev-with-express/introduction-to-express/README.md:15]:
```suggestion
Now, to install the Express.js in this application, run the following command:
```

File: backend-dev-with-express/introduction-to-express/README.md (hunk 1 above)
Comment (vikispeaks) [reply backend-dev-with-express/introduction-to-express/README.md:8]: Also, let's use `Express.js` is all locations where it is referenced.

File: backend-dev-with-express/introduction-to-express/README.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+const express = require('express');
+const app = express();
+```
+Here, the first line requires the **Express** module that was installed via NPM, and the second line sets up our Express application. With this application (`app`), you can configure and add functionality to the server.
+
+Then,
+```js
+app.listen(3000)
+```
+The `app.listen()` function tells the server to start listening for connections on a particular port, in this case port `3000`. This is why we went to `localhost:3000` to look at our **Hello World** example.
```
Comment (vikispeaks) [inline backend-dev-with-express/introduction-to-express/README.md:56]:
```suggestion
The `app.listen()` function tells the server to start listening for connections on a particular port, in this case, port `3000`. This is why we went to `localhost:3000` to look at our **Hello World** example.
```

**Review by vikispeaks — CHANGES_REQUESTED** [review]

File: backend-dev-with-express/introduction-to-express/README.md (hunk 1 above)
Comment (avishekjana) [reply backend-dev-with-express/introduction-to-express/README.md:8]: OK

**Pull request merged**